routing/async_router.cpp

```diff
@@ -227,6 +230,15 @@ void AsyncRouter::LogCode(IRouter::ResultCode code, double const elapsedSec)
     case IRouter::IntermediatePointNotFound:
       LOG(LWARNING, ("Can't find intermediate point node"));
       break;
+    case IRouter::TransitRouteNotFoundNoNetwork:
+      LOG(LWARNING, ("Transit route not found because no transit network in route point mwm"));
```

Comment:
no transit network is found?
because there is no transit network?

Reply:
No transit route is found because there's no transit network in mwm of route point

Reply:
https://github.com/mapsme/omim/pull/8000